src/filters/Seudo2D.js

```diff
@@ -3,13 +3,14 @@
  */
 
 
-var Bruker = require("/home/abol/git/brukerconverter/src/brukerconverter");
-
+var convertZip = require("/home/abol/git/brukerconverter/src/brukerconverter").convertZip;
```

Comment:
Don't use absolute paths. Use the relative path to src instead:  var converZip = require("./brukerconverter")